www/templates/html/FeedList.tpl.php

```diff
@@ -47,13 +47,26 @@
 
 <div class="wdn-band">
     <div class="wdn-inner-wrapper wdn-inner-padding-none">
-        <p class="mh-sort-filter">
-            <span class="mh-btn-group">
-                <a href="/channels/" class="wdn-button wdn-button-brand <?php echo (($context->options['orderby'] != 'datecreated') && ($context->options['orderby'] != 'plays')) ? ' active' : '' ?>">Alphabetical</a>
-                <a href="?orderby=datecreated&amp;order=DESC" class="wdn-button wdn-button-brand <?php echo ($context->options['orderby'] == 'datecreated') ? ' active' : '' ?>">Recent</a>
-                <a href="?orderby=plays&amp;order=DESC" class="wdn-button wdn-button-brand <?php echo ($context->options['orderby'] == 'plays') ? ' active' : '' ?>">Popular</a>
-            </span>
-        </p>
+        <?php
+        $buttons = new stdClass();
```

Comment:
As a best practice, I would _not_ instantiate a `stdClass`. I consider it an internal PHP class that happens to be usable from userland. Instead, I would use an array and _cast_ it using `(object) $buttons`.